src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -18,6 +18,13 @@
         "see https://www.overleaf.com/help/342 for more information."
     );
 
+    public static final List<String> OVERLEAF_V1_DEPRECATED_REASON = Arrays.asList(
+            "Overleaf v1 is deprecated, and you need to migrate this project to v2.",
+            "",
+            "If this is unexpected, please contact us at support@overleaf.com, or",
+            "see https://www.overleaf.com/help/342 for more information."
```

Comment:
This wording is bad. 

Suggestions @jdleesmiller and @jpallen ?

Reply:
How about:

> This project needs to be migrated to Overleaf v2. You can do this by visiting the project at:
> 
> https://www.overleaf.com/<project_id>
>
> After migrating this project to Overleaf v2, you will be prompted to update your git remote to the new Overleaf v2 identifier.
>
> Please see https://www.overleaf.com/help/342 for more information, or contact us at support@overleaf.com.

Reply:
We could mirror the wording from https://github.com/sharelatex/web-sharelatex/blob/master/app/views/project/v2-import.pug .

> This project has not yet been moved into the new version of Overleaf. You will need to move it in order to continue working on it. Please visit the project online to do this:

Ah, James beat me to it. James's wording also looks good to me.

Reply:
I like John's better (consistent and less harsh), so the whole thing would be:

> This project has not yet been moved into the new version of Overleaf. You will need to move it in order to continue working on it.  Please visit the project online to do this:
> 
> https://www.overleaf.com/<project_id>
> 
> After migrating this project to Overleaf v2, you will be prompted to update your git remote to the new Overleaf v2 identifier.
> 
> Please see https://www.overleaf.com/help/342 for more information, or contact us at support@overleaf.com.